ods_ci/tests/Tests/1300__model_registry/Model_Registry_Testing.md

```diff
@@ -20,6 +20,8 @@ To review and run it locally you will need the following:
   ```bash
   ./odstest --install-identity-providers
   ```
+  Note: once you install the identity providers in might take upto 15 mins to become active.
```

Comment:
Hm, is this truly true? I think that in our Jenkins jobs we wait just 3 minutes :thinking: but can't really argue here as I never ever checked the code.

Reply:
I don't really know Jan, I had a bit of a wait when I was running locally. It turned out though I had caching issues when trying to log back in so this might be wilding over the mark.  If you are sure it is less than that I'd be happy to go with your suggestion.  Maybe 5 mins at the outside?